src/main/java/Problem3/Problem3.java

```diff
@@ -8,9 +8,20 @@
 
 public class Problem3 {
     public static void printListInReverse(ListNode list, PrintStream print) {
-        // homework
-        // !!! must use your ArrayStack or LinkedListStack in problem 1
-        // print with the following format. System.out.println() will not pass test.
-        print.println("hello world");   // place holder
+
+        LinkedListStack<ListNode> data = new LinkedListStack<>();
```

Comment:
arguably, putting ListNode in the stack is a bit overkill. Just put the integer values would be good enough.